synapse/tests/test_types.py

```diff
@@ -282,6 +282,21 @@ def test_type_pend(self):
         self.true(tlib.addType('bar', subof='int'))
         self.nn(tlib.getDataType('foo'))
 
+    def test_type_gettdef(self):
+        tlib = s_types.TypeLib()
+
+        tnfo = tlib.getTypeDef('int')
+        self.nn(int)
```

Comment:
Should this be `tnfo`? the `int` class is not going to be None

Reply:
good catch

Reply:
fixed